src/js/components/awardv2/idv/federalAccounts/FederalAccountsSummary.jsx

```diff
@@ -30,7 +30,7 @@ const FederalAccountsSummary = ({
                     </span>
                 </div>
                 <div className="award-funding-summary__data">
-                    <span>Total Count Of Funding Agencies</span>
+                    <span>Total Count of Funding Agencies</span>
```

Comment:
👌

Reply:
I think there is actually a ticket for this as well?

Reply:
not that I am aware of. I also checked with Melissa.